### ISSUE TITLE
Error on https://www.addictinggames.com/action/n-game

### ISSUE THREAD

**Issue by matt-ag** [ruffle-rs/ruffle#7709]
# Error Info
Error name: Error
Error message: panicked at 'Stage must always have a root movie', core/src/display_object/stage.rs:576:14
Error stack:
```
Error: panicked at 'Stage must always have a root movie', core/src/display_object/stage.rs:576:14
    at V.e.wbg.__wbg_new_df6e6ab7a65c4c4d (https://www.addictinggames.com/sites/all/themes/starkish/build/js/ruffle/core.ruffle.e0e876bcfd237f27a772.js:1:6065)
    at std::sync::once::Once::call_once::{{closure}}::h96c2b4473ff6d536 (wasm://wasm/0160c3f6:wasm-function[1205]:0x2b1618)
    at std::sync::once::Once::call_inner::h1c414562e2619117 (wasm://wasm/0160c3f6:wasm-function[1377]:0x2e6810)
    at core::ops::function::FnOnce::call_once{{vtable.shim}}::h14edbfd8adf99b9a (wasm://wasm/0160c3f6:wasm-function[4677]:0x42b6f9)
    at std::panicking::rust_panic_with_hook::h1c368a27f9b0afe1 (wasm://wasm/0160c3f6:wasm-function[3067]:0x3dc417)
    at core::panicking::panic_fmt::hcdb13a4b2416cf82 (wasm://wasm/0160c3f6:wasm-function[4859]:0x43180f)
    at core::option::expect_failed::h9f179d8645f7b768 (wasm://wasm/0160c3f6:wasm-function[5470]:0x43eab0)
    at ruffle_core::display_object::stage::Stage::root_clip::ha5c2687f2091a1a5 (wasm://wasm/0160c3f6:wasm-function[3394]:0x3f24de)
    at ruffle_web::Ruffle::tick::h80b89f3b236277b4 (wasm://wasm/0160c3f6:wasm-function[609]:0x1bfbae)
    at ruffle_web::Ruffle::new_internal::{{closure}}::{{closure}}::{{closure}}::h62d65489931f7892 (wasm://wasm/0160c3f6:wasm-function[5647]:0x446adf)
```

# Player Info
Allows script access: true
Player type: Object
SWF URL: /sites/default/files/flash_games/n-game.swf
Param menu: false
Param allowscriptaccess: always
Param allownetworking: all
Param movie: /sites/default/files/flash_games/n-game.swf
Attribute 0: undefined
Attribute 1: undefined
Attribute 2: undefined
Attribute 3: undefined
Attribute 4: undefined

# Page Info
Page URL: https://www.addictinggames.com/action/n-game
SWF URL: https://www.addictinggames.com/sites/default/files/flash_games/n-game.swf

# Browser Info
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/104.0.0.0 Safari/537.36
Platform: Win32
Has touch support: false

# Ruffle Info
Version: 0.1.0
Name: nightly 2022-07-25
Channel: nightly
Built: 2022-07-25T00:17:23.429Z
Commit: afb9ecc4ab8a6f6b9b295ddcb029a4ca80230a9e
Is extension: false

**Comment by Lord-McSweeney** [comment]
Will need `*` property access on `XMLList` after #10026.

**Comment by Dinnerbone** [comment]
Panic is fixed in #10026
